pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -103,3 +103,8 @@ def configure_progress_bar(self, refresh_rate=1, process_position=0):
 
     def _trainer_has_checkpoint_callbacks(self):
         return len(self.trainer.checkpoint_callbacks) > 0
+
+    def attach_model_logging_functions(self, model):
+        for callback in self.trainer.callbacks:
+            callback.log = model.log
+            callback.log_dict = model.log_dict
```

Comment:
Question: should we also allow for callbacks to write predictions aka `self.write_prediction` and `self.write_prediction_dict`?

Reply:
Yes, but within another PR :)